datajoint/external.py

```diff
@@ -338,7 +338,7 @@ def delete(self, *, delete_external_files=None, limit=None, display_progress=Tru
             error_list = []
             for uuid, external_path in items:
                 try:
-                    count = (self & {'hash': uuid}).delete_quick(get_count=True)  # optimize
+                    count = len(self & {'hash': uuid})  # optimize
```

Comment:
Why do we need to get the count? Why do we need an extra query rather than deleting directly?